src/components/streamSelect/StreamSelect.js

```diff
@@ -12,7 +12,8 @@ class StreamSelect extends Component {
     ];
     this.state = {
       currentStream: "5ddb98f5e4b0937e6a4507f2",
-      customStreamID: ""
+      customStreamID: "",
+      customLatencyField: "",
```

Comment:
Please be consistent. If you wish to distinguish between values of inputs as opposed to player option values then I'd suffix not just this state property (I generally use Input to match the HTML tag and function of the values) and I don't think you need to prefix with custom anymore, e.g: streamIdInput and targetLatencyInput.

PS: I generally also prefer to camelCase abbreviations (fooBarWarg instead of fooBARWARG), but if within React or this template they use the other way, then follow that convention.